pam/pam.go

```diff
@@ -75,7 +78,11 @@ func (h *pamModule) Authenticate(mTx pam.ModuleTransaction, flags pam.Flags, arg
 	client, closeConn, err := newClient(args)
 	if err != nil {
 		log.Debug(context.TODO(), err)
-		return pam.ErrAuthinfoUnavail
+		msgErr := showPamMessage(mTx, pam.ErrorMsg, err.Error())
+		if msgErr != nil {
```

Comment:
I would avoid having another error which has `msg` (which sounds like it’s a string).
Please, make it only line then and rename the error variable to something that doesn’t conflict with `err`.

Reply:
Right, done.